src/main/java/org/sasanlabs/service/vulnerability/jwt/JWTVulnerability.java

```diff
@@ -662,4 +669,51 @@ private ResponseEntity<GenericVulnerabilityResponseBean<String>> getJWTResponseB
                         true, token, true, CollectionUtils.toMultiValueMap(headers));
         return responseEntity;
     }
+
+    @AttackVector(
+            vulnerabilityExposed = VulnerabilityType.HEADER_INJECTION,
+            description = "HEADER_INJECTION_VULNERABILITY_EXAMPLE")
+    @VulnerableAppRequestMapping(
+            value = LevelConstants.LEVEL_13,
+            htmlTemplate = "LEVEL_13/HeaderInjection_Level13")
+    public ResponseEntity<GenericVulnerabilityResponseBean<String>> getHeaderInjectionVulnerability(
+            HttpServletRequest request) {
+        String jwtToken = request.getHeader("Authorization");
+        if (jwtToken == null || !jwtToken.startsWith("Bearer ")) {
+            return new ResponseEntity<>(
+                    new GenericVulnerabilityResponseBean<>("No JWT token provided", true),
+                    HttpStatus.BAD_REQUEST);
+        }
+
+        jwtToken = jwtToken.substring(7); // Remove "Bearer " prefix
```

Comment:
I think it is better to replace bearer from token instead of index substring.

Reply:
Thanks, I have implemented the improvement